tests/system/cli/test_tmux_tools.py

```
@@ -94,32 +94,56 @@ def _run_tmux(self, *args: str, check: bool = True) -> str:
-            last_output = output
-            if any(line.lstrip().startswith("│ >") for line in output.splitlines()):
+            if self.ready_file.is_file():
                 return
             time.sleep(STARTUP_POLL_INTERVAL_SECONDS)
 
         raise TimeoutError(
-            "Timed out waiting for tunacode tmux session to render the editor prompt.\n"
-            f"Last capture:\n{last_output or self.capture()}"
+            "Timed out waiting for tunacode tmux session readiness file.\n"
+            f"{self._startup_diagnostics()}"
+        )
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)